.github/workflows/rules.yml

```diff
@@ -2,55 +2,23 @@
 name: Tests
 
 on:
+  workflow_dispatch:
   push:
   pull_request:
     types: [labeled]
 
-
 jobs:
   build:
-    if: contains(github.event.pull_request.labels.*.name, 'run-workflow') || github.event_name == 'push'
+    #if: contains(github.event.pull_request.labels.*.name, 'run-workflow') || github.event_name == 'push'
```

Comment:
If this is for debugging purpose, add a TODO comment to remove before merging

Reply:
I commented this line to let the workflow dispatch work. Now this if condition is restored and the workflow dispatch trigger deleted.